tests/tests.rs

```diff
@@ -465,33 +465,66 @@ fn rgb_to_yuv_size(num_planes: u32, width: u32, height: u32) {
     }
 }
 
-fn yuv_to_rgb_size_mode_stride(
+fn yuv_to_rgb_size_format_mode_stride(
     num_planes: u32,
     width: u32,
     height: u32,
     color_space: ColorSpace,
-    luma_stride: usize,
-    chroma_stride: usize,
-    dst_stride: usize,
+    luma_fill_bytes: usize,
+    u_chroma_fill_bytes: usize,
+    v_chroma_fill_bytes: usize,
+    dst_fill_bytes: usize,
+    format: PixelFormat,
 ) {
     let w = width as usize;
     let h = height as usize;
 
-    let chroma_height = h / 2;
-    let in_size = (luma_stride * h) + (chroma_stride * chroma_height);
+    let luma_stride = w + luma_fill_bytes;
+    let chroma_height = match format {
+        PixelFormat::Nv12 | PixelFormat::I420 => h / 2,
+        _ => {
+            println!("Unsupported pixel format");
+            0
```

Comment:
See https://github.com/aws/dcv-color-primitives/pull/5/files#r372458354 for all function definition